article/app/views/fragments/articleHeaderGarnett.scala.html

```diff
@@ -22,7 +22,7 @@
         <div class="content__header tonal__header">
 
             <div class="u-cf">
-                @if(article.content.tags.tags.exists(_.id == "tone/news")) {
+                @if(article.content.tags.tags.exists(_.id == "tone/news") || article.content.tags.tags.exists(_.id == "commentisfree/commentisfree") {
```

Comment:
Missing the second closing bracket for `@if()` on all of these - hence the build failure.

Reply:
🙏 thanks for spotting this!